include/swift/Option/FrontendOptions.td

```diff
@@ -223,7 +223,7 @@ def swift_module_file
 
 def explict_swift_module_map
```

Comment:
You will need to correct this selector name as well.

Reply:
Done! Thanks, for the catch!